### PR TITLE
Fix href link content in the view case component.

### PR DESCRIPTION
**Pull request by axmb**

### CODE REVIEW

File: verification/curator-service/ui/src/components/ViewCase.test.tsx [hunk 1]
```diff
@@ -77,8 +90,15 @@ it('loads and displays case', async () => {
     // Transmission.
     expect(getByText(/Vector borne/)).toBeInTheDocument();
     expect(getByText(/Gym/)).toBeInTheDocument();
-    expect(getByText(/bbf8e943dfe6e00030892dcc/)).toBeInTheDocument();
-    expect(getByText(/aaf8e943dfe6e00030892dee/)).toBeInTheDocument();
+    expect(getByText(/bbf8e943dfe6e00030892dcc/)).toHaveAttribute(
+        'href',
+        '/cases/view/bbf8e943dfe6e00030892dcc',
```
[the remark this answers is not in the document]
Comment (axmb) [reply verification/curator-service/ui/src/components/ViewCase.test.tsx:95]: I must be looking in the wrong place -- are you referring [to this](https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/verification/curator-service/ui/src/components/ViewCase.tsx#L387)?

[the remark this answers is not in the document]
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/ViewCase.test.tsx:95]: Yes, I didn't notice this link when I removed the URL route for viewing cases - I should've removed it then. That link should either be removed now, or we should add a link for viewing cases that opens up the modal.

[the remark this answers is not in the document]
Comment (axmb) [reply verification/curator-service/ui/src/components/ViewCase.test.tsx:95]: Got it. I'm not super clear on the intended behavior -- but I'm happy to wait on this and merge if you'd like, just LMK.

**Comment by axmb** [comment]
[Bug fixes](https://trello.com/c/DKJztzhL/276-bug-fixes)

File: verification/curator-service/ui/src/components/ViewCase.test.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/ViewCase.test.tsx:95]: It's not a regression on this PR, so I'm okay for you to submit and we can fix later.

**Pull request merged**